### PR TITLE
Law regs redesign

### PR DESCRIPTION
**Pull request by kewitham**
This PR updates the law and regs landing page and design standards landing page and adds a regulations landing page based on this [figma](https://www.figma.com/file/2BtQC1OVFFWm1bVLCxW7HL/ADA.gov---2023?type=design&node-id=3940-3920&mode=design&t=Bekkinem8nNYsDgh-0).
Note that there are still some outstanding questions so this should not be merged.
Screenshot 2023-12-29 at 1 06 36 PM
Screenshot 2023-12-29 at 1 07 10 PM
Screenshot 2023-12-29 at 1 07 39 PM

### CODE REVIEW

File: _includes/landing/law-reg-tiles.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <h3>
+                    <a href="{{'/law-and-regs/regulations/' | relative_url}}" class="law-reg-tile-link" data-ga-event-name="Learn more regulations">Regulations</a>
+                </h3>
+                <div style="margin-top: 0px" class="no-bullets padding-left-2">
+                <p>DOJ is responsible for issuing regulations under the Americans with Disabilities Act (ADA) that explain the rights of people with disabilities and the obligations of those covered by the laws.</p>
+                </div>
+                <a href="{{'/law-and-regs/regulations/' | relative_url}}" class="usa-button law-reg-tile-link" data-ga-event-name="Learn more regulations">Learn more</a>
+            </div>
+        </div>
+    </div>
```
Comment (okkays) [inline _includes/landing/law-reg-tiles.html:15]: These don't appear to be scaling well to mobile:

image

**Review by okkays — CHANGES_REQUESTED** [review]
Code LGTM!

I'd just double check how things work on mobile

**Comment by vwilliamsn** [comment]
@kewitham - Here are some of my suggested edits based on the standards subpage screenshot that @sanchez-amanda provided late yesterday:

•	I think it would be helpful to include hyperlinks so that folks can access these documents in multiple places on this page. See here for example:  “[1991 ADA Standards](https://www.ada.gov/1991standards/1991standards-archive.html)” and adopted revised ADA Standards on September 15, 2010, known as the “[2010 ADA Standards](https://www.ada.gov/regs2010/2010ADAStandards/2010ADAstandards.htm)”, along with the [Guidance on the 2010 Standards](https://www.ada.gov/regs2010/2010ADAStandards/Guidance2010ADAstandards.htm).  

•	Remove the last sentence in the new paragraph under Versions of the ADA Standards – “In 2011, the Department issued…..”  as it’s repetitive with a sentence in the next paragraph.
 
Happy to discuss at tomorrow’s meeting if you have questions.

**Comment by vwilliamsn** [comment]
@kewitham for all of the newly redesigned pages please update the left side navigation:

•	In the side navigation it should be “Title II Regulation” (singular) and “Title III Regulation” (singular).

cc: @sanchez-amanda

**Comment by vwilliamsn** [comment]
@kewitham @sanchez-amanda - in the hamburger feature is there a way to show that the Law, Regs & Standards is a landing page and the ADA, Regulations & Standards are subpages?

Also, can we make changes to the hamburger feature, left navigation, landing page title and info boxes so that it reads: _Law, Regulations & Standards._

Thanks!

**Comment by kewitham** [comment]
@sanchez-amanda @vwilliamsn made the updates based on your feedback - let me know if I missed anything!

**Comment by vwilliamsn** [comment]
Great job, @kewitham! I took a quick glance and spotted a couple of places that need a minor tweak or two. I'll take a deeper look tomorrow.

Change requested for: 1) all the Info boxes should read 'Law" (singular – see image below) & 2) add hyperlink to Guidance on the 2010 Standards in the paragraph under Versions of the ADA Standards (see image below)


image
image

 cc: @sanchez-amanda

**Comment by vwilliamsn** [comment]
> @sanchez-amanda @vwilliamsn made the updates based on your feedback - let me know if I missed anything!

LGTM! Thanks @kewitham. I'm going to provide the federalist link to leadership for their review. cc: @sanchez-amanda

**Comment by vwilliamsn** [comment]
@kewitham, I received a request to make changes to the heading (below)
image

Please change it to: Proposed Changes to Our Regulations.

cc: @sanchez-amanda

**Comment by vwilliamsn** [comment]
@kewitham - here are some edits from leadership. We are still holding for final approval.

1) [Leadership requested changes to homepage for Law, Regs and Standards.docx](https://github.com/usdoj-crt/beta-ada/files/14088699/Leadership.requested.changes.to.homepage.for.Law.Regs.and.Standards.docx)

Info line recommendation - Consider adding images to the Reg subpage and make the images clickable (for consistency with the design standards page).
t2-reg-cover
t3-reg-cover

**Comment by kewitham** [comment]
Closing this out as the changes now exist in a new [PR](https://github.com/usdoj-crt/beta-ada/pull/748) here to deal with conflicts.

**Pull request closed without merging**